rubin_sim/maf/metrics/weakLensingSystematicsMetric.py

```diff
@@ -116,3 +116,87 @@ def run(self, dataSlice, slicePoint):
             return self.badval
         nvisits = len(np.where(dataSlice[self.expTimeCol] > self.min_expTime)[0])
         return nvisits
+
+
+class RIZDetectionCoaddExposureTime(BaseMetric):
+    """A metric computing the total exposure time of an riz coadd.
+
+    This metric is intended to be used as a proxy for depth fluctuations in catalogs detected
+    from coadds of the r, i and z bands together. This coadding + detection scheme is used by
+    metadetection (weak lensing shear estimator) and will likely be adopted by the Rubin science
+    pipelines.
+
+    It counts the total exposure time in all three bands, excluding dusty regions, exposures
+    that are too short, or areas where not all bands ugrizY are present. We do not make a depth
+    cut explicitly since that is circular (and thus confuses my feeble mind :/).
+
+    TODO:
+     - apply some sort of inverse variance weighting to the coadd based on sky level?
+     - use some sort of effective exposure time that accounts for the PSF?
```

Comment:
Right. The final weight of each image in the cell-based coadd will be something like an inverse variance weight based on the sky background pixel noise. If we move to Kaiser coadds, this weight will get even more complicated. I'm thinking now that instead of dealing with all of this uncertainty, it is simpler to count raw exposure time with the expectation that some effects go one way (bad seeing makes it worse) and others go the other (better weighting in the coadding reduces the effects of bad exposures).